docs/guide/repository.md

```diff
@@ -484,109 +472,14 @@ ezpublish:
     system:
         # "site_group" refers to the group configured in site access
         site_group:
-            # "default" refers to the cache pool, the one configured on stash.caches above
+            # "default" refers to the cache pool, the one configured above
             cache_pool_name: "default"
```

Comment:
this has changed: https://github.com/ezsystems/ezplatform/blob/2.0/app/config/ezplatform.yml#L27

It refers to simple service now, so there full access to create whatever you want, and hence not abstracted and limited by the config supported, as was the case with Stash Bundle.